static/js/chart.js

```diff
@@ -205,6 +205,11 @@ export function createChart(chartSelector, taskTypes, taskStatus, tasks) {
   var timeDomainEnd = d3.time.year.offset(tasks[tasks.length - 1].endDate, +1);
   var height = taskTypes.length * rowHeight;
   var width = document.querySelector(chartSelector).clientWidth - margin.right - margin.left;
+  var tickSize = 1;
+
+  if (window.innerWidth < 1000) {
```

Comment:
I would suggest increasing the threshold as the years get very close.

Reply:
Thanks. Done